app/frontend/stylesheets/config/_setup.scss

```diff
@@ -22,6 +28,13 @@ body {
   min-height: 100vh;
 }
 
+
+h1, h2, h3 {
+h1, h2 {
```

Comment:
This does not make sense. h1 and h2 are not nested inside h1 and h2 in any html.erb-document I guess :)

Reply:
And you already have the same thing at line 38->40 too

Reply:
I didn't write these lines of code. I know that they dont make sense but i dont know how the got there

Reply:
Probably when merging master into your branch

Reply:
I just deleted line 32 to 36. 38 to 40 is making the job i believe